<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialog</class>
 <widget class="QDialog" name="dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>590</width>
    <height>341</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icon/res.qrc">
    <normaloff>:/case/add_case.png</normaloff>:/case/add_case.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="4" column="1">
      <widget class="QLineEdit" name="edt_caseSeq">
       <property name="placeholderText">
        <string>#</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="2">
          <widget class="QComboBox" name="cb_caseType">
           <item>
            <property name="text">
             <string>Transaction</string>
            </property>
            <property name="icon">
             <iconset resource="icon/res.qrc">
              <normaloff>:/case/tr_test.png</normaloff>:/case/tr_test.png</iconset>
            </property>
           </item>
           <item>
            <property name="text">
             <string>UI</string>
            </property>
            <property name="icon">
             <iconset resource="icon/res.qrc">
              <normaloff>:/case/ui_test.png</normaloff>:/case/ui_test.png</iconset>
            </property>
           </item>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLineEdit" name="edt_category">
           <property name="text">
            <string/>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="placeholderText">
            <string>Category 선택</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QToolButton" name="btn_category">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="icon/res.qrc">
             <normaloff>:/case/open_folder.png</normaloff>:/case/open_folder.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Description</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QTextEdit" name="edt_caseDesc">
       <property name="placeholderText">
        <string>Case 설명</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Case Name</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="edt_caseNm">
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Ex) T지원금 + 분할상환</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Category</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Seq</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>edt_category</tabstop>
  <tabstop>edt_caseNm</tabstop>
  <tabstop>edt_caseDesc</tabstop>
  <tabstop>edt_caseSeq</tabstop>
  <tabstop>btn_category</tabstop>
  <tabstop>cb_caseType</tabstop>
 </tabstops>
 <resources>
  <include location="icon/res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>331</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>331</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>edt_category</sender>
   <signal>returnPressed()</signal>
   <receiver>btn_category</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>313</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel">
     <x>469</x>
     <y>23</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
